Makefile

```diff
@@ -14,7 +14,8 @@ E2E_SPEC ?=
 
 UI ?=
 
-RELEASE_VERSION := $(shell grep -v '^\#' ./release-version)
+# RELEASE_VERSION := $(shell grep -v '^\#' ./release-version)
+RELEASE_VERSION ?= $(shell git describe --tags --dirty --always)
```

Comment:
https://github.com/pingcap/tidb-dashboard/pull/1627/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L40

The line above can be removed since we use git tag as `RELEASE_VERSION` and `-fips` will appear in git tag.

Reply:
ok, let me check.